plaso/parsers/dsv_parser.py

```diff
@@ -59,6 +62,7 @@ def __init__(self, encoding=None):
     """
     super(DSVParser, self).__init__()
     self._encoding = encoding
+    self._maximum_line_length = len(self.COLUMNS) * self.FIELD_SIZE_LIMIT
```

Comment:
Shouldn't this be:
`( len(self.COLUMNS) * ( self.FIELD_SIZE_LIMIT + length of field separator) ) +  length of end of line` ?